modules/tile-converter/src/i3s-converter/helpers/geometry-converter.d.ts

```diff
@@ -28,7 +28,8 @@ export default function convertB3dmToI3sGeometry(
   attributeStorageInfo: any,
```

Comment:
I didn't realize there were still .js and .d.ts files in the loaders.gl code base. Can we move to .ts or is that not possible for tile-converter?

I find it inconvenient to review the .js files below because there are no typings in them.